Dockerfile

```
@@ -0,0 +1,25 @@
+# Runtime image - JAR is pre-built in CI
+FROM eclipse-temurin:21-jre-alpine
+
+WORKDIR /app
+
+# 타임존 설정 및 wget 설치 (health check용)
+RUN apk add --no-cache tzdata wget && \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)